fetools-app/src/pages/ColorPicker.jsx

```diff
@@ -37,5 +47,19 @@ export default function ColorPicker() {
 
     )
 
+
+  function handleOnPageLoad(){
+    console.log(searchParams.get('color')!==null?true:false)
+    if(searchParams.get('color')!==null){
+      navigate(`/color-picker?color=${encodeURIComponent(searchParams.get('color'))}`)
+      return
+    } 
+  }
+
+  function handleQuery(newColor){
+    //e.preventDefault()
+    setSearchParams({color: newColor})
+  }
```

Comment:
```suggestion
  function handleQuery(color){
    color = color.slice(1);
    setSearchParams({ color });
  }
```

Let's keep the url a little cleaner by leaving off the `#` for our color code parameter.